cartoframes/auth/credentials.py

```diff
@@ -40,6 +40,7 @@ class Credentials:
             documentation
             <http://docs.python-requests.org/en/master/user/advanced/>`__
             for more information.
+        allow_non_secure (bool, optional): Allow non secure http connections.
```

Comment:
Add in the doc: "By default is disabled", or something like that.

Reply:
doc updated